.env

```diff
@@ -98,4 +98,15 @@ _APP_VCS_GITHUB_CLIENT_SECRET=
 _APP_VCS_GITHUB_WEBHOOK_SECRET=
 _APP_MIGRATIONS_FIREBASE_CLIENT_ID=
 _APP_MIGRATIONS_FIREBASE_CLIENT_SECRET=
-_APP_ASSISTANT_OPENAI_API_KEY=
\ No newline at end of file
+_APP_ASSISTANT_OPENAI_API_KEY=
+_APP_MESSAGE_SMS_PROVIDER_MSG91_SENDER_ID=
```

Comment:
I prefer DSN structure too

Reply:
Yea DSN would be worth considering here. 

We could do
* sms://username:password@host/path?paramA=valueA
* email://username:password@host/path?paramA=valueA

and so on .. 

this can then be accessed with 
```php
$dsn->getScheme() # sms
$dsn->getHost() # host
$dsn->getUser() #username
$dsn->getPassword() #password
$dsn->getPath() # path
$dsn->getParam('paramA') # valueA
```

Reply:
These env variables are only for e2e tests of messaging endpoints. Since users configure the providers, we store the credentials directly in our db data.

Reply:
Just a note here: These env variables are only for e2e tests of messaging endpoints. Since users configure the providers, we store the credentials directly in our db data.

Reply:
This has been changed to 

- _APP_MESSAGE_SMS_TEST_DSN=
- _APP_MESSAGE_EMAIL_TEST_DSN=
- _APP_MESSAGE_PUSH_TEST_DSN=